README.md

```diff
@@ -130,6 +115,18 @@ In order to interact with GCP from your system you will need to authenticate:
 ```console
 gcloud auth application-default login
 ```
+### Enable GCP APIs
```

Comment:
create.sh is already enabling these APIs. We don't need the user to manually enable them

Reply:
My hope is the user will never, ever, touch bash. I'm still working on that part of the docs and making final decisions. This is for the automated path.

Reply:
I take it back. I still have them generating variables with a script, but otherwise I have them running the individual commands so that they understand what's going on.